diff --git a/android/src/main/java/com/reactnativeandroidwidget/oss/CSSBackgroundDrawable.java b/android/src/main/java/com/reactnativeandroidwidget/oss/CSSBackgroundDrawable.java
index bb254b1c448ff813c248282ce0ce9f91dd56ed7a..7aef482175324e71fdce9bc390558d7d55b5cbfd 100644
--- a/android/src/main/java/com/reactnativeandroidwidget/oss/CSSBackgroundDrawable.java
+++ b/android/src/main/java/com/reactnativeandroidwidget/oss/CSSBackgroundDrawable.java
@@ -1457,7 +1457,7 @@ public class CSSBackgroundDrawable extends Drawable {
         Shader compositeShader = null;
         for (BackgroundImageLayer backgroundImageLayer : mBackgroundImageLayers) {
             Rect bounds = getBounds();
-            Shader currentShader = backgroundImageLayer.getShader(bounds.width(), bounds.height());
+            Shader currentShader = backgroundImageLayer.getShader(bounds);
             if (currentShader == null) {
                 continue;
             }
